docs/src/pages/blog/2025-12-31-similar-transactions/og-image

```diff
@@ -0,0 +1,162 @@
+#!/usr/bin/env node
+
+import fs from 'node:fs';
+import { createRequire } from 'node:module';
+import path from 'node:path';
+import { Resvg } from '@resvg/resvg-js';
+import satori from 'satori';
+
+
+const require = createRequire(import.meta.url);
+
+const WIDTH = 1200;
+const HEIGHT = 630;
+
+async function generateOg() {
+  // TODO Make this work with woff2 files.
+  const fontPath = require.resolve('@fontsource-variable/inter/files/inter-latin-standard-normal.woff2');
+  // TODO Since it doesn't work with woff2 files, just use the downloaded files from fontsource for now.
+  const fontData = fs.readFileSync('/home/elliotcourant/Downloads/inter_5.2.8/ttf/inter-latin-400-normal.ttf');
```

Comment:
I'll come back and fix this later:

- [ ] Make this work with fontsource packages or if needed download the ttf files via CMake as part of the dependency target.
- [ ] Make this script not specific to this blog post, instead have it parse the frontmatter for every page and generate og images if one is not outright specified.
- [ ] Clean this up?